schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE animals (
+  id INTEGER PRIMARY KEY,
+  name VARCHAR(100),
+  date_of_birth DATE,
+  escape_attempts INTEGER,
+  neutered BOOLEAN,
+  weight_kg DECIMAL(5,2),
+  species VARCHAR(50)
```

Comment:
- [ ] At this point of the project, the animal table had already been created, what was expected from you is to alter the table by adding a column using the `ALTER TABLE animal ADD species varchar(255);` 

Kindly add the column using the ALTER command instead of adding it like you did in line 8.

Reply:
> * [x]  At this point of the project, the animal table had already been created, what was expected from you is to alter the table by adding a column using the `ALTER TABLE animal ADD species varchar(255);`
> 
> Kindly add the column using the ALTER command instead of adding it as you did in line 8.

- [ ] You have implemented nearly all of the suggestions made by the previous reviewer. However, it appears that you may have overlooked one recommendation, which was to create a new column named `species` with a data type of string, as requested, using the `alter` command. :+1: